Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -225,16 +160,14 @@ class BrowserLocationView : UIView, UIGestureRecognizerDelegate, UITextFieldDele
         }
     }
 
-    func SELlongPressLocation(recognizer: UILongPressGestureRecognizer) {
-        if recognizer.state == UIGestureRecognizerState.Began {
-            delegate?.browserLocationViewDidLongPressLocation(self)
-        }
+    func textFieldShouldBeginEditing(textField: UITextField) -> Bool {
+        delegate?.browserLocationViewDidTapLocation(self)
```

Comment:
This could fire if editing was started some way besides tapping the text field? Seems like it could be confusing some day.